src/javascript/app/pages/trade/digit_trade.js

```diff
@@ -38,12 +41,20 @@ const DigitDisplay = (() => {
         $container
             .addClass('normal-font')
             .html($('<h5 />', { text: contract.display_name, class: 'center-text' }))
-            .append($('<div />', { class: 'gr-8 gr-centered gr-12-m' })
+            .append($('<div />', { class: 'gr-8 gr-centered gr-12-m', style: `height: ${calculated_height}px;` })
                 .append($('<div />', { class: 'gr-row', id: 'table_digits' })
                     .append($('<strong />', { class: 'gr-3', text: localize('Tick') }))
                     .append($('<strong />', { class: 'gr-3', text: localize('Spot') }))
                     .append($('<strong />', { class: 'gr-6', text: localize('Spot Time (GMT)') }))))
             .append($('<div />', { class: 'digit-ticker invisible', id: 'digit_ticker_container' }));
+        $digit_table = $('#table_digits');
+        $digit_table
+            .append($('<div />', { class: 'barspinner dark' })
+                .append($('<div />', { class: 'rect1' }))
+                .append($('<div />', { class: 'rect2' }))
+                .append($('<div />', { class: 'rect3' }))
+                .append($('<div />', { class: 'rect4' }))
+                .append($('<div />', { class: 'rect5' })));
```

Comment:
is it worth moving the adding of a spinner to a utility function and passing `$digit_table` to it?

Reply:
Yeah, move it to it's own function.